components/login/LoginForm.tsx

```diff
@@ -20,6 +20,8 @@ import Link from "next/link";
 import { useRouter } from 'next/navigation';
 import { useState } from "react";
 import { z } from "zod";
+import { AiFillEye, AiFillEyeInvisible } from 'react-icons/ai';
```

Comment:
Hey @pratyushsingha , we use [this ](https://lucide.dev/icons)icon library and wants to maintain this as much as possible.

I hope there is similar icon like this in this library too. do the necessary changes:)